test/schema.sql

```diff
@@ -32,7 +32,6 @@ CREATE TABLE proposals (
   choices JSON NOT NULL,
   start INT(11) NOT NULL,
   end INT(11) NOT NULL,
-  delegation INT(1) NOT NULL,
```

Comment:
Is there some other way to use schema from hub 🙈  we didn't add many fields here from there recently 🙈

Reply:
What you have in mind ? github actions to sync the file with hub ?

Reply:
Excellent Idea! I think we can check for changes on each pull request sequencer and sync with the hub